lua/plenary/profile.lua

```diff
@@ -0,0 +1,11 @@
+local profile = {}
```

Comment:
Any other functions to add while we're here?

Reply:
Fine for now.